pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -200,6 +200,9 @@ type JaegerIngressOpenShiftSpec struct {
 
 	// +optional
 	DelegateURLs string `json:"delegate-urls,omitempty"`
+
+	// +optional
+	HtpasswdFile string `json:"htpasswd-file,omitempty"`
```

Comment:
Just realised, this is using a hyphen (same for `delegate-urls`) whereas the convention we started to use was lower camel case.